packages/api-explorer/src/utils/path.ts

```diff
@@ -28,6 +28,21 @@ import type { ApiModel, IMethod, IType } from '@looker/sdk-codegen'
 import { firstMethodRef } from '@looker/sdk-codegen'
 import type { Location as HLocation } from 'history'
 
+/**
+ * Builds a path matching the route used by scene with search parameters
+ * @param specKey A string to identify the spec in the URL
+ * @param tag Corresponding tag
+ * @param name A method or type name
+ * @param params Hash of query param name/value pairs to include in the destination url
+ * @returns a path including search parameters
+ */
+export const buildNavigationPath = (
```

Comment:
I manually added this, should `path.d.ts` also be manually updated to include this method's type definition?